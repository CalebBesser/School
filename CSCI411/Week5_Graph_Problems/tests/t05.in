6 8
0 1
1 2
1 3
2 1
2 5
3 0
4 2
5 4
printGraph
isFlower
