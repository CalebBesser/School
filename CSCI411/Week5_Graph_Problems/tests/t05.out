Adjacency lists of the graph are:
0: 1 
1: 2 3 
2: 1 5 
3: 0 
4: 2 
5: 4 
The graph is NOT a flower-graph.
